docs/FAQ.md

```diff
@@ -265,6 +266,8 @@ Some studies include data from one or more targeted sequencing platforms which d
 ## Other pages
 ### Does the Mutual Exclusivity tab calculate its statistics using all samples/alterations or only a specific subset?
 The calculations on the Mutual Exclusivity tab are performed using all samples included in the query. A sample is defined as altered or unaltered for each gene based on the [OQL](https://www.cbioportal.org/onco_query_lang_desc.jsp) utilized in the query - by default, this will be non-synonymous mutations, fusions, amplifications and deep deletions.
+### What are the values of the box and whiskers in a boxplot?
+In boxplots on cBioPortal, the box is drawn from the 25th percentile (Q1) to the 75th percentile (Q3), with the horizontal line in between representing the median. Whiskers are drawn independently above and below the box, and will extend to the maximum or minimum data values, unless there are outlier values, in which case the whisker will extend to 1.5 * IQR (interquartile range = Q3-Q1). Outliers are defined as values that extend beyond 1.5 * IQR.
```

Comment:
@tmazor did we find some URL to a reference for this that demonstrates this is a common way to draw boxplots? E.g. http://www.physics.csbsju.edu/stats/box2.html, https://towardsdatascience.com/understanding-boxplots-5e2df7bcbd51 or https://en.wikipedia.org/wiki/Box_plot ?

Reply:
Maybe for the diehard boxplotters a link to the code? https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/lib/boxPlotUtils.ts